packages/editor/package.json

```diff
@@ -41,6 +41,7 @@
 		"@wordpress/compose": "file:../compose",
 		"@wordpress/core-data": "file:../core-data",
 		"@wordpress/data": "file:../data",
+		"@wordpress/data-controls": "file:../data-controls",
```

Comment:
What is this for?

Reply:
The IDE automatically added a wrong import the line was removed.